packages/core-utils/src/time.js

```diff
@@ -88,34 +82,27 @@ export function formatSecondsAfterMidnight(seconds, timeFormat) {
  */
 export function getUserTimezone() {
   if (process.env.NODE_ENV === "test") return process.env.TZ;
-  // FIXME There is an issue with tz.guess being undefined that has not yet been
-  // resolved. https://github.com/opentripplanner/otp-ui/issues/152
-  if (!moment.tz || typeof moment.tz.guess !== "function") {
-    // eslint-disable-next-line no-console
-    console.warn(
-      "Error guessing user's timezone (moment.tz or moment.tz.guess not defined). Defaulting to America/New_York."
-    );
-    return "America/New_York";
-  }
-  return moment.tz.guess();
+  return Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
There's a very small chance that browser compatibility might be an issue here.

Reply:
See https://github.com/opentripplanner/otp-ui/pull/271/files#r681344828

Reply:
Note added to readme in 55e76870